### ISSUE TITLE
Tried to install with "make install" but...

### ISSUE THREAD

**Issue by lumizila** [jordansissel/xdotool#207]
I got this following error on running "make install":
xdo.c:24:10: fatal error: 'X11/Xlib.h' file not found

Could it be because xdtool is not meant for OSx?

**Comment by jordansissel** [comment]
> Could it be because xdotool is not meant for OSx?

Correct. xdotool will only work on systems running X11. X11 is a graphical system usually found on Unix/Linux operating systems.
